### PR TITLE
Bug fixes and cleanup

### PR DESCRIPTION
**Pull request by hampuskraft**
[Deleted for privacy reasons]

### CODE REVIEW

File: GearBot/database/DatabaseConnector.py [hunk 1]
```diff
@@ -1,8 +1,13 @@
+import warnings
+
 from tortoise.models import Model
 from tortoise import fields, Tortoise
 
 from Util import Configuration, GearbotLogging
 
+# https://github.com/aio-libs/aiomysql/issues/103
+warnings.filterwarnings("ignore", module=r"aiomysql")
```
Comment (AEnterprise) [inline GearBot/database/DatabaseConnector.py]: looking at the issue this is only for getting rid of mysql warnings when making the tables? if so this should only be applied to that code and not global and forever

File: requirements.txt [hunk 2]
```diff
@@ -3,6 +3,7 @@ tortoise-orm==0.16.17
 Pillow==7.2.0
 requests==2.24.0
 aioredis==1.3.1
+aiomysql==0.1.1
```
Comment (AEnterprise) [inline requirements.txt]: this should be done in another profile (or however it is called in python land). while gearbot supports mysql i highly advise against using it as it is very bad at handling the messages at high volumes

**Review by AEnterprise — CHANGES_REQUESTED** [review]
thanks but i do have some remarks about the changes for mysql (something i don't really recommend using anyways. its caused way too many outages for me already compared to postgres)

some might also conflict with some local changes i made but appearently haven't pushed yet but i'll take care of those after

**Comment by hampuskraft** [comment]
@AEnterprise Thanks for reviewing, I've omitted the changes for aiomysql, makes sense that PostgreSQL is preferred.

**Pull request merged**